### DISCUSSION TITLE
Projects beta - Iteration Burndown

### DISCUSSION THREAD

**Discussion by mlynam** [community/community#7886] [Projects and Issues]
It would be nice to have a burndown chart or at the very least, a progress bar for issue completion like with milestones or the old style Projects.

**Comment by fmagarrotebridge** [comment]
gráfico de burndown seria um diferencial bem grande

**Comment by maku77** [comment]
We will switch from Jira to Projects once burndown is supported.

**Comment by joachimweyl** [comment]
+1
